README.md

```diff
@@ -69,6 +69,9 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 15 (Early 2020) Base              |  1532:0255  |
 | Razer Huntsman Mini                           |  1532:0257  |
 | Razer Blade Stealth (Late 2020)               |  1532:0259  |
+| Razer BlackWidow V3 PRO (USB)                 |  1532:025A  |
+| Razer BlackWidow V3 PRO (Bluetooth)           |  1532:025B  |
+| Razer BlackWidow V3 PRO (WIFI)                |  1532:025C  |
```

Comment:
From looking at the product page it doesn't look like it's using Wifi (which would be weird anyways). The product page says this, so maybe use this wording?
* Wired
* 2.4 Ghz Wireless
* Bluetooth

Also please fix the product name to have the correct capitalization (in all files): `Razer BlackWidow V3 Pro`

Reply:
This still seems to be an open issue

> Also please fix the product name to have the correct capitalization (in all files): Razer BlackWidow V3 Pro